packages/devtools-network-console/src/actions/locale-action.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { i18n } from 'network-console-shared';
+import { AnyAction } from 'redux';
+import { ThunkAction } from 'redux-thunk';
+import { IView } from 'store';
+
+export interface ISetLocaleAction {
+    type: 'SET_LOCALE';
+    locale: string;
+}
+
+export function makeSetLocaleAction(locale: string): ISetLocaleAction {
+    return {
+        type: 'SET_LOCALE',
+        locale,
+    };
+}
+
+export function initializeLocaleDictionary(dictionary: any, locale: string): ThunkAction<void, IView, void, AnyAction> {
+    return async dispatch => {
+        i18n.loadLocalization(dictionary, locale);
+        dispatch(makeSetLocaleAction(locale));
+    };
+}
+
+export type LocaleAction = 
```

Comment:
Yes, it's a convention I've used throughout whenever I alias an action type union. Multiple types make it clear why it would be useful:

```ts
export type EnvironmentAction =
    ISetEnvironmentAuthorizationAction |
    ISetEnvironmentVariablesAction |
    IClearEnvironmentVariablesAction
    ;
```

It allows the semicolon to avoid being part of the diff in the future.